commercetools-models/src/main/java/io/sphere/sdk/categories/Category.java

```diff
@@ -173,6 +174,10 @@
     @Nullable
     CustomFields getCustom();
 
+    @Nonnull
```

Comment:
Okay just saw that Michael used it here for Products... I wonder why. But Products is not auto-generated and I don't see anything else problematic here, so give it a try.